src/t_set.c

```diff
@@ -849,8 +849,17 @@ int qsortCompareSetsByRevCardinality(const void *s1, const void *s2) {
     return 0;
 }
 
+/* SINTER / SINTERSTORE / SINTERCARD
+ *
+ * 'cardinality_only' work for SINTERCARD, only return the cardinality
+ * with minimum processing and memory overheads.
+ *
+ * 'limit' work for SINTERCARD, stop searching after reaching the limit.
+ * Passing a 0 means unlimited.
+ */
 void sinterGenericCommand(client *c, robj **setkeys,
-                          unsigned long setnum, robj *dstkey, int cardinality_only) {
+                          unsigned long setnum, robj *dstkey, int cardinality_only,
+                          unsigned long limit) {
```

Comment:
```suggestion
                          unsigned long setnum, robj *dstkey,
                          int cardinality_only, unsigned long limit) {
```
Not sure if I'm the only one who thinks this will align better.

Reply:
yes, i thought so too and decided not to comment.

Reply:
yes... after a resee, a bad one